src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetOverlappingTrainingDateRequest/GetOverlappingTrainingDateRequestQueryResult.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Application.Queries.GetOverlappingTrainingDateRequest
+{
+    public class GetOverlappingTrainingDateRequestQueryResult
+    {
+        public long Id { get; set; }
+        public long DraftApprenticeshipId { get; set; }
+        public long PreviousApprenticeshipId { get; set; }
+        public OverlappingTrainingDateRequestResolutionType? ResolutionType { get; set; }
+        public OverlappingTrainingDateRequestStatus Status { get; set; }
+        public OverlappingTrainingDateRequestEmployerAction? EmployerAction { get; set; }
```

Comment:
This can be removed as we no longer have employerAction in the db

Reply:
Done